src/main/java/org/zendesk/client/v2/Zendesk.java

```diff
@@ -2720,6 +2735,19 @@ public Iterable<Holiday> getHolidaysForSchedule(Long scheduleId) {
             handleList(Holiday.class, "holidays")));
     }
 
+    public MediaResponse getGuideMedias(Map<String, Object> filterParameters, String pageAfter, int perPage, Sorting sorting) {
+        Map<String, Object> pageParameters = new HashMap<>();
+        pageParameters.put("after", pageAfter);
+        pageParameters.put("size", perPage);
+
+        TemplateUri tmpl = tmpl("/guide/medias{?filter*,page*,sort}")
```

Comment:
It should be `filter[name]=...`, `page[after]=...` etc. Are you sure this code will produce `filter[name]` query params?

Reply:
Probably this magic will happen in `setGroupParameters`.
But I believe a simple unit test will be the best here. Confirm that the string will be properly formated and also confirm your concern about parameters escaping.

Reply:
there is a test for group parameters in TemplateUriTest, also I tested it with real environment in RealSmokeTest.